core/pages/Category.js

```diff
@@ -12,14 +12,15 @@ import Composite from '@vue-storefront/core/mixins/composite'
 import { Logger } from '@vue-storefront/core/lib/logger'
 import { mapGetters, mapActions } from 'vuex'
 import onBottomScroll from '@vue-storefront/core/mixins/onBottomScroll'
+import themeConfig from 'theme/theme.config'
```

Comment:
you cannot assume you have access to theme config here. Someone can implement theme without theme config and it would be a breaking change for them

Btw. isn't Category.js not used anymore? Everything is in `Category.vue` in theme as I recall. Wouldn't change that file.